tests/test_pytorch_dataloader_api.py

```diff
@@ -46,9 +46,13 @@ def forward(self, x):
     "workers",
     [1, 2, 3],
 )
+@pytest.mark.parametrize(
+    "num_of_attributes",
+    [1],
```

Comment:
Why do we test only just for 1 number of attributes?

Reply:
Multiple attribute support for PyTorch is not yet implemented. I just had to make some minor changes to the PyTorch reader tests because of the common data ingestion utility with Tensorflow readers.